media-fonts/ocr/ocr-0.2.recipe

```diff
@@ -0,0 +1,48 @@
+SUMMARY="Free optical character recognition fonts from the Tsukurimashou Project"
+DESCRIPTION="This package contains freely-usable fonts for the OCR A and OCR B typefaces.
+OCR-A is the font used for the ISBNs on books; and OCR-B is used for the human-readable digits \
+on UPC/EAN bar codes. A book with a standard bar code label uses both.
+
+I translated existing Metafont sources to MetaType1 to create the enhanced TrueType, PostScript, \
+and OpenType versions included in this package.
+They are available for free use, including in commercial applications; my own contributions are \
+public domain, and others have licensed their copyright claims."
+HOMEPAGE="https://tsukurimashou.osdn.jp/index.php.en#ocrb"
+COPYRIGHT="2012 Matthew Skala"
+LICENSE="Public Domain"
+REVISION="1"
+SOURCE_URI="https://jaist.dl.sourceforge.jp/tsukurimashou/56948/ocr-$portVersion.zip"
+CHECKSUM_SHA256="39289c641520265ecedbade99f01600f316f8196ec57f71c8402d3ba09438666"
+ADDITIONAL_FILES="fontconfig-reload-cache.sh"
+
+ARCHITECTURES="any"
+DISABLE_SOURCE_PACKAGE=yes
+
+POST_INSTALL_SCRIPTS="$relativePostInstallDir/fontconfig-reload-cache.sh"
+
+PROVIDES="
+	ocr = $portVersion
+	"
+REQUIRES="
+	"
+
+BUILD_REQUIRES=""
+BUILD_PREREQUIRES=""
+
+BUILD()
+{
+	true
+}
```

Comment:
```suggestion
```

Reply:
Had that thought too, wasn't sure :)

Reply:
Done, PS using git_x86-2.36.1 atm here, so far working OK :)